x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/onboarding/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { AssetInventoryOnboarding } from './asset_inventory_onboarding';
```

Comment:
i would also remove this one, especially since `asset_inventory_onboarding` is essentially the index file but the folder is named `onboarding` ( i would also rename, but your call )